recipes/ncempy/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "ncempy" %}
+{% set version = "1.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: fd08bb09baf467b69c64983993ce133ca21cc9b6f3294ceffbab95110ab3c062
+  patches:
+    - long_description.diff
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+  entry_points:
+    - ncem2png = ncempy.command_line.ncem2png:main
+
+requirements:
+  host:
+    - python >=3
+    - pip
+    - setuptools
+  run:
+    - python >=3
+    - numpy
+    - scipy
+    - matplotlib
+    - h5py
+    - setuptools
+
+test:
+  imports:
+    - ncempy
```

Can you also test the command?
```suggestion
    - ncempy
  commands:
    - ncem2png --help
```